### ISSUE TITLE
observer.com

### ISSUE THREAD

**Issue by ricktendo** [reek/anti-adblock-killer#2961]
http://observer.com/2017/03/nsa-no-evidence-obama-donald-trump-tower-wiretapping-hoax/

Mozilla Firefox
Greasemonkey
Adblock Plus
EasyList + AakList + Adblock Warning Removal List

![observer](https://cloud.githubusercontent.com/assets/10519232/23976645/ac7a5fb0-09ae-11e7-8779-683d18c8d6ed.PNG)

**Comment by uBlock-user** [comment]
Cannot reproduce.

add - `||$third-party,script,domain=observer.com` and `@@||wp.com^,domain=observer.com`

**Comment by ulmont** [comment]
I can reproduce in Chrome with the same subscriptions.  after adding the filters above the warning is gone, but so is all of the actual content on the page as well.

**Comment by drachs** [comment]
Yep, it happened to me to today.   I didn't try the filter because of ulmon't report that it was taking out page content.
